lib/cc/cli/prepare/quality.rb

```diff
@@ -0,0 +1,59 @@
+require "safe_yaml/load"
+
+module CC
+  module CLI
+    class Prepare
+      class Quality < Command
+        ENGINES_CONFIG = {
+          "complexity-ruby" => {
+            "enabled" => true,
+            "channel" => "beta",
+          },
+          "duplication" => {
+            "enabled" => true,
+            "channel" => "cronopio",
+          },
+        }.freeze
+
+        def execute
+          Dir.chdir(CC::Analyzer::MountedPath.code.container_path) do
+            content =
+              if (existing_contents = read_codeclimate_yml)
+                modify(existing_contents)
+              else
+                { "engines" => ENGINES_CONFIG }
+              end
+
+            write_codeclimate_yml(content)
+          end
+        end
+
+        private
+
+        def modify(content)
+          content.delete("ratings")
+          content["engines"] ||= {}
+          content["engines"].merge!(ENGINES_CONFIG)
```

Comment:
I think this is fine. One thing to note that we might be losing for the Duplication engine is some additional configuration, like engine-specific exclude paths and a reduced language set.